Dockerfile

```diff
@@ -1,49 +1,58 @@
-FROM node:10.3-alpine
-LABEL maintainer="https://github.com/verdaccio/verdaccio"
+FROM node:10.3-alpine as builder
 
-RUN apk --no-cache add openssl && \
-    wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 && \
-    chmod +x /usr/local/bin/dumb-init && \
-    apk del openssl && \
-    apk --no-cache add ca-certificates wget && \
+ENV NODE_ENV=production \
+    VERDACCIO_BUILD_REGISTRY=https://registry.npmjs.org/
+
+RUN apk --no-cache add openssl ca-certificates wget && \
```

Comment:
I think there is no need to keep `openssl` here for building as we install later in final the final image (line 37). Can we remove it?

Also, we are installing `ca-certificates` here but in final image we don't. Don't we need `ca-certificates` in final docker image?

Reply:
About these packages I was not sure so I tried to leave them as they were. There's another comment asking for the openssl one in the final image; regarding ca-certificates, I think they might be needed because of the downloaded keys for glibc, but unsure.

I took a look in case I could also replace that glibc apk download by a normal install. Apparently it was added because there's no flow native package in alpine, but seems [there might be one soon](https://github.com/facebook/flow/issues/1605#issuecomment-403242700).